src/diffpy/utils/diffraction_objects.py

```diff
@@ -418,12 +418,33 @@ def _get_original_array(self):
             return self.on_d(), "d"
 
     def on_q(self):
+        """Return the tuple of two 1D numpy arrays containing q and y data.
+
+        Returns
+        -------
+        (xarray, yarray) : tuple of ndarray
```

Comment:
let's change `xarray` to `q-array` to be explicit.  And also below, xarray-> `tth-array` and so on.  Just to make it really clear.